2.0/README.md

```diff
@@ -0,0 +1,500 @@
+# Vector Tile Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in
+this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## 1. Purpose
+
+This document specifies a space-efficient encoding format for tiled geographic vector data. It is designed to be used in browsers or server-side applications for fast rendering or lookups of feature data.
+
+## 2. File Format
+
+The Vector Tile format uses [Google Protocol Buffers](https://developers.google.com/protocol-buffers/) as a encoding format. Protocol Buffers are a language-neutral, platform-neutral extensible mechanism for serializing structured data.
+
+## 2.1. File Extension
+
+The filename extension for Vector Tile files SHOULD be `mvt`. For example, a file might be named `vector.mvt`.
+
+## 2.2. Multipurpose Internet Mail Extensions (MIME)
+
+When serving Vector Tiles the MIME type SHOULD be `application/vnd.mvt`.
+
+## 3. Projection and Bounds
+
+A Vector Tile represents data based on a square extent within a projection. A Vector Tile SHOULD NOT contain information about its bounds and projection. The file format assumes that the decoder knows the bounds and projection of a Vector Tile before decoding it.
```

Comment:
Rather than SHOULD NOT (imposing a requirement) this seems to be a statement of fact which would be "A Vector Tile does not contain information..."